src/Catalyst.Core.Lib/CoreLibProvider.cs

```diff
@@ -164,5 +166,11 @@ protected override void Load(ContainerBuilder builder)
 
             base.Load(builder);
         }
+
+        // TODO: rethink validation of transaction signature
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4585091)